cli/standalone/virtual_fs.rs

```diff
@@ -266,17 +266,20 @@ impl VfsBuilder {
 
     let dir = self.add_dir(path.parent().unwrap())?;
     let name = path.file_name().unwrap().to_string_lossy();
-    let data_len = data.len();
+    let offset_and_len = OffsetWithLength {
+      offset,
+      len: data.len() as u64,
+    };
     match dir.entries.binary_search_by(|e| e.name().cmp(&name)) {
       Ok(index) => {
         let entry = &mut dir.entries[index];
         match entry {
           VfsEntry::File(virtual_file) => match sub_data_kind {
             VfsFileSubDataKind::Raw => {
-              virtual_file.offset = offset;
+              virtual_file.offset = offset_and_len;
             }
             VfsFileSubDataKind::ModuleGraph => {
-              virtual_file.module_graph_offset = offset;
+              virtual_file.module_graph_offset = offset_and_len;
```

Comment:
A test case I added surfaced a bug where we weren't storing the length of module graph only files meaning a file could be corrupted.